huawei-cloud/dashboard/pom.xml

```diff
@@ -47,5 +47,9 @@
             <groupId>com.netflix.hystrix</groupId>
             <artifactId>hystrix-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.servicecomb</groupId>
+            <artifactId>dashboard-client</artifactId>
+        </dependency>
     </dependencies>
 </project>
```

Comment:
better to add new line on end

Reply:
I checked most xml files do not add a new line in this project.  And no new line for xml files do not have [a clear problem](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file) .